.github/workflows/health_checks.yml

```diff
@@ -10,6 +10,8 @@ on:
       - main
       - hotfix
       - feature/**
+  schedule:
+    - cron: '0 12 * * *' # Every day 12:00 UTC
```

Comment:
This will run in early AM pacific time. 
I.e. test results will be available before work day starts.
Additional benefit is that this schedule won't compete with devs running their workflow.